cdhweb/pages/templates/cdhpages/landing_page.html

```diff
@@ -12,7 +12,15 @@
 {% endblock %}
 
 {% block content-header %}
-<header {% if page.header_image %}style="background-image:url('{{ MEDIA_URL }}{{ page.header_image }}')"
+{% comment %}
+image asset guidelines for landing page
+L,M @1x W: 2560px H: 680px, @2x W: 5120px H: 1360px
+S @1x W: 736px H: 320px, @2x W: 1472px H: 640px
```

Comment:
fwiw the L/M 2x seems huge; I don't think we ever want to be serving stuff larger than 2500px. but fine for now; we can always revisit.

Reply:
These are Xinyi's guidelines, maybe they need to be revisited. I agree it's huge (and I think what we've always been serving, even for mobile — although I guess before we just served out whatever we had; will wagtail make images larger?). This is why I asked you about the background image and different sizes.

Do you want to make a new issue for addressing this and add it to 3.1 ? It would be a good improvement to the site.

Reply:
created #266 and added to 3.1